packages/trader/src/Modules/Trading/Containers/trade-params-mobile.jsx

```diff
@@ -101,7 +101,7 @@ const TradeParamsModal = observer(({ is_open, toggleModal }) => {
                 header={<div />}
                 disableApp={disableApp}
                 toggleModal={toggleModal}
-                height='auto'
+                height={is_vanilla ? '53.8rem' : 'auto'}
```

Comment:
Why do we have to hardcode this?

Reply:
i suspect it's temporary because the implementation is only for Vanillas at the moment, later on it will be done for all trade types, is that right @kate-deriv ? )

Reply:
Correct, thank you @maryia-deriv . It's temporary until decision of applying same changes to all contracts

Reply:
My question was more of why do we need to hardcore this exact value here? Is there no better way to calculate the value and why auto doesn't work for this case?

Reply:
The height of Container with `height:auto` depends on the blocks inside it. So, it happens, that height will jump if the amount of included blocks changes.
According to the new design Container should have fixed height in order not to 'jump' and disturb user.